core/storage/classifier/gae_models_test.py

```diff
@@ -87,9 +87,9 @@ def test_query_new_and_pending_training_jobs(self):
             [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
             'state_name2', feconf.TRAINING_JOB_STATUS_FAILED, 1)
 
-        training_jobs, cursor, more = (
+        training_jobs, offset = (
             classifier_models.ClassifierTrainingJobModel.
-            query_new_and_pending_training_jobs(None))
+            query_new_and_pending_training_jobs())
```

Comment:
It would also be valuable to test that the offset is utilized when it's passed in (rather than defaulting to 0).

Reply:
Done